dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -660,9 +658,9 @@ public static void main(String[] argv) {
      */
     private void serverMetadata() {
         try {
-            final Metadata metadata = ga4ghApi.metadataGet();
+            final TRSService serviceinfo = ga4ghv20Api.getServiceInfo();
             final Gson gson = io.cwl.avro.CWL.getTypeSafeCWLToolDocument();
```

Comment:
This is an interesting way to do it, getting a CWL-specific Gson for non-CWL, but was like that and no need to change. :)